test/integration/irohad_test.cpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2018 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <boost/process.hpp>
+#include "gtest/gtest.h"
+
+using namespace boost::process;
+using namespace std::chrono_literals;
+
+const std::string path_irohad(PATHIROHAD);
+const std::string irohad_executable = path_irohad + "/irohad";
+const std::string path_example = path_irohad + "/../../example";
```

Comment:
Not really sure it's a good idea to depend on the examples

Reply:
I do not think it will be better to hardcode all these files in a test body.

Reply:
It's possible to just copy it, don't see any problem on that

Reply:
But to copy them I still have to know their location, right? Sorry but I haven't got the point.

Reply:
I mean persistently copy, e.g by unix `cp`. Or maybe make symlink

Reply:
I see. But to do ```cp``` I should know what to copy and where to copy from - that's the same dependency as before.
Or do you mean to make a copy not in the test code but just copy files in repository and push them after? Anyway I do not like this idea because we will need to update 2 copies of same files.

Reply:
> Or do you mean to make a copy not in the test code but just copy files in repository and push them after

Exactly

Reply:
I am really afraid of their possible desynchronisation.